File: LinkedList.c
#include <stdio.h>
#include <stdlib.h>

int main() {
    struct node{
        int data;
        struct node *next;
    };
    
    struct node *new1;
    new1 = (struct node *)malloc(sizeof(struct node));
    new1->data = 10;
    new1->next = NULL;
    
    struct node *new2;
    new2 = (struct node *)(malloc(sizeof(struct node)));
    new2->data = 20;
    new2->next = NULL;
    
    new1->next = new2;

    printf("Node 1 data: %d\n", new1->data);
    printf("Node 2 data: %d\n", new2->data);

    return 0;
}

File: LinkedList.py
class Node:
    def __init__(self, data):
        self.data = data
        self.next = None

def main():
    new1 = Node(10)
    new2 = Node(20)

    new1.next = new2
    print(f"Node 1 data: {new1.data}")
    print(f"Node 2 data: {new2.data}")

if __name__ == "__main__":
    main()
